stages/eib_image

```diff
@@ -103,7 +103,8 @@ if [[ "${EIB_IMAGE_SDBOOT}" = "true" ]]; then
 fi
 
 kargs=(--karg=rw --karg=splash \
-    --karg=plymouth.ignore-serial-consoles --karg=quiet --karg=loglevel=0)
+    --karg=plymouth.ignore-serial-consoles --karg=quiet --karg=loglevel=0 \
+    --karg=systemd.firstboot=no)
```

Comment:
I'd like to understand why this is useful/necessary compared to eos6.
eos6 also seems to have this systemd firstboot service, not sure if it runs or not.
Perhaps it doesn't because our ostree builder creates /etc/machine-id as an empty file?

Reply:
Ok. I can take a look at changing this next week.

Reply:
FWIW, @wjt has done some deep investigation here in the past. See https://phabricator.endlessm.com/T35527. That also includes some of my "set root password on first boot" toying I was doing. That also includes another reason why GNOME OS has `systemd.firstboot=no` on the command line - it ensures that systemd-firstboot runs if the `ConditionFirstBoot` conditions are met but makes it run non-interactively.

Anyways, the conclusion of that task was to leave Endless the way it is for now.

Reply:
@ssssam this is still pending - or are you thinking of doing the machine-id bit as a separate task?

Reply:
I'd rather do this separately. Opened https://github.com/endlessm/eos-build-meta/issues/95.